client/blocks/reader-full-post/style.scss

```diff
@@ -140,15 +107,13 @@
 
 .reader-full-post__story {
 	max-width: 720px;
+	padding: 0 24px;
```

Comment:
Re: gutter padding. I got rid of the conflicting sources of padding introduced on different elements at different breakpoints and it should be consolidated here. Feel free to adjust it, but we also need to adjust the breakpoint to hide the sidebar along with this.